x-pack/plugin/watcher/src/internalClusterTest/java/org/elasticsearch/xpack/watcher/actions/email/EmailAttachmentTests.java

```diff
@@ -205,8 +205,15 @@ public void testThatEmailAttachmentsAreSent() throws Exception {
                     throw e;
                 }
             }
-            assertNotNull(searchResponse);
-            assertHitCount(searchResponse, 1);
+            try {
```

Comment:
I guess you could just move the try down one line and save. the `if` in the `finally` :) If it's null we don't need to release it and don't even get to the second line.

Reply:
Done! I updated also `WatchMetadataTests` that had a similar try-finally block